README.md

```diff
@@ -68,22 +69,34 @@ Official:
 
 ## Addons
 
-### Streams
+### Movie & series
+
+Torrents:
 
 - [1337x torrents](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/1337x-torrents): Torrent addon for 1337x
 - [Deflix](https://github.com/doingodswork/deflix-stremio): Looks up your selected movie on YTS, The Pirate Bay, 1337x and ibit and automatically turns your selected torrent into a debrid/cached stream, for high speed and **no P2P uploading**
-- [dlive.tv](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/dlive): Live streams from dlive.tv
 - [HorribleSubs](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/horriblesubs): Anime torrents
 - [Juan Carlos 2](https://github.com/JCB9090/juan-carlos-torrents-2): Torrent addon for KAT.cr and torrentz.eu
-- [OpenSubtitles (*official* addon)](https://github.com/Stremio/stremio-opensubtitles): Adds subtitles from [OpenSubtitles](https://www.opensubtitles.org) to your stream
 - [pct](https://github.com/JCB9090/pct-addon): Torrent addon for EZTV and YTS
 - [Piratebay](https://github.com/ThanosDi/piratebay-stremio-addon): Torrent addon for The Pirate Bay
-- [podcasts](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/podcasts)
-- [podcasts-for-all](https://github.com/NivM1/podcasts-for-all)
 - [RARBG torrents](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/rarbg-torrents): Torrent addon for RARBG
 - [rarbg](https://github.com/sebastiencs/rarbg-addon): Torrent addon for RARBG
 - [Stream Quality Filter (aka "SQF")](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/stream-quality-filter): Fetches streams from RARBG, 1337x, YTS and TPB+ addons, removes duplicates and sorts them by quality
 
+### TV channels
+
+Livestreams:
+
+- [dlive.tv](https://github.com/sleeyax/stremio-addons/tree/master/packages/addons/dlive): Live streams from dlive.tv
+
+Podcasts:
```

Comment:
Could this be its own header? It doesn't fit into `TV channels` very well.

Reply:
Hmm depends how deep you want to nest the categories... 

We could add `TV channels` as a root category and add `podcasts` and `livestreams` etc. as subcategories like this:
```
### TV channels
#### Livestreams
#### Podcasts
```
Going this route the content table might become a bit messy if you also want to include these subcategories:
```
4. [Addons](#addons)
    1. [Movies & TV shows](#Movies--TV-shows)
    2. [TV channels](#TV-channels)
        * [Podcasts](#podcasts)
        * [Livestreams](#livestreams)
```

Another option is to separate everything: `TV channels` becomes a category and `livestreams` and `podcasts` become categories too. No subcategories. This is probably the better option, right?

Reply:
Maybe the deeper nesting or more 3rd level headers make sense when there are more list elements, and for now we can put it in a "Others" 3rd level header? But let's do that either later or in another PR, I'll accept this one already because it improves things already.